roles/prometheus/templates/alert-manager.yaml

```diff
@@ -0,0 +1,168 @@
+---
+# Source: prometheus/templates/alertmanager-configmap.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: prometheus
+    release: prometheus-{{ k8s_prometheus_image_tag }}
+    component: "alertmanager"
+  namespace: {{ k8s_prometheus_namespace }}
+  name: prometheus-alertmanager
+data:
+  alertmanager.yml: |-
+    global:
+       slack_api_url: '{{ prometheus_slack_api_url }}'
+
+    receivers:
+      - name: default-receiver
+        slack_configs:
+          - channel: '{{ prometheus_slack_channel }}'
+            send_resolved: true
+
+    route:
+      group_wait: 10s
+      group_interval: 5m
+      receiver: default-receiver
+      repeat_interval: 3h
+---
+# Source: prometheus/templates/alertmanager-serviceaccount.yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app: prometheus
+    release: prometheus-{{ k8s_prometheus_image_tag }}
+    component: "alertmanager"
+  namespace: {{ k8s_prometheus_namespace }}
+  name: prometheus-alertmanager
+---
+# Source: prometheus/templates/alertmanager-clusterrolebinding.yaml
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    app: prometheus
+    release: prometheus-{{ k8s_prometheus_image_tag }}
+    component: "alertmanager"
+  namespace: {{ k8s_prometheus_namespace }}
```

Comment:
ClusterRoleBinding can not have any namespace

Reply:
fixed